qt/scientific_interfaces/Indirect/IndirectDataManipulationTab.h

```diff
@@ -0,0 +1,76 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2013 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "IndirectInstrumentConfig.h"
+#include "IndirectPlotOptionsPresenter.h"
+#include "IndirectTab.h"
+
+// Suppress a warning coming out of code that isn't ours
```

Comment:
do we still need all of these?

Reply:
I am not sure if they are still needed, I could not find the origional issue it fixed, I figured it would be better the leave it instead of remove it and hope nothing poped up.